vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -90,7 +90,9 @@ if ($windowsSku -eq "2019") {
             "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.23.11", # for k8s 1.23.x
             "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.24.3", # for k8s 1.24.x
             # OMS-Agent (Azure monitor). Owner: ganga1980 (Ganga Mahesh Siddem)
-            "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-ciprod01182023-095c864a")
+            "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-ciprod01182023-095c864a",
+            # CNS (Container Networking Service). Owner: jaer-tsun (Jaeryn)
+            "mcr.microsoft.com/containernetworking/azure-cns:v1.4.44")
```

Comment:
Will azure-cns v1.4.44 be scheduled to WS2019 nodes? NOTE: In k8s 1.23, we still support users to create Windows agent pools with both docker and containerd.

Reply:
yup the new CNS will also deploy to windows 2019 for CNI overlay scenario